src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  time: string;
+};
+// eslint-disable-next-line
+export class Clock extends Component<Props, State> {
+  intervalId = 0;
+
+  state: Readonly<State> = {
+    time: new Date().toUTCString().slice(-12, -4),
```

Comment:
will be better to create separate function for date transforming, as you repeated this code twice